File: python/strings.py
name = "John"
age = 23

# 1. Old-style formatting (C-style)
print("My name is %s and I am %d years old." % (name, age))

# 2. str.format() method
print("My name is {} and I am {} years old.".format(name, age))

# 3. f-string (Python 3.6+)
print(f"My name is {name} and I am {age} years old.")

File: cpp/strings.cpp
// cpp/strings.cpp
#include<iostream>
#include<string>
std::string name = "John";

int main(){
    const char* name = "John";
    int age = 23;

    //1. C-style printf formatting (closest to Python's % operator)
    std::printf("My name is %s and I am %d years old.\n", name, age);

    //2. C++ style with cout
    std::cout << "My name is " << name << " and I am " << age << " years old.\n";

    return 0;
}